pkg/storage/snapshot/snapshot.go

```diff
@@ -628,6 +628,8 @@ func (ctrl *VMSnapshotController) getSnapshotPVC(namespace, volumeName string) (
 
 	if volumeSnapshotClass != "" {
 		return pvc, nil
+	} else {
```

Comment:
you dont need else since you return from the if

Reply:
No problem, I thought it would be more explicit. Removed the else.